lib/modules/manager/vendir/artifacts.spec.ts

```diff
@@ -0,0 +1,356 @@
+import { mockDeep } from 'jest-mock-extended';
+import { join } from 'upath';
+import { envMock, mockExecAll } from '../../../../test/exec-util';
+import { Fixtures } from '../../../../test/fixtures';
+import { env, fs, git, partial } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import type { StatusResult } from '../../../util/git/types';
+import type { UpdateArtifactsConfig } from '../types';
+import * as vendir from '.';
+
+jest.mock('../../datasource', () => mockDeep());
+jest.mock('../../../util/exec/env');
+jest.mock('../../../util/http');
+jest.mock('../../../util/fs');
+jest.mock('../../../util/git');
+
+const adminConfig: RepoGlobalConfig = {
+  localDir: join('/tmp/github/some/repo'), // `join` fixes Windows CI
+  cacheDir: join('/tmp/renovate/cache'),
+  containerbaseDir: join('/tmp/renovate/cache/containerbase'),
+};
+
+const config: UpdateArtifactsConfig = {};
+const vendirLockFile1 = Fixtures.get('vendir_1.lock');
+const vendirLockFile2 = Fixtures.get('vendir_2.lock');
+const vendirFile = Fixtures.get('vendir.yml');
+
+describe('modules/manager/vendir/artifacts', () => {
+  beforeEach(() => {
+    env.getChildProcessEnv.mockReturnValue(envMock.basic);
+    GlobalConfig.set(adminConfig);
+  });
+
+  afterEach(() => {
+    GlobalConfig.reset();
+  });
```

Comment:
remove, set in before will overwrite old config

Reply:
removed